docs/ReleaseProcess.md

```diff
@@ -25,41 +27,69 @@ eBPF for Windows, and how to service it later.
     Version number updated to '0.12.0' in D:\work\ebpf-for-windows\scripts\..\installer\Product.wxs
     Rebuilding the solution, please wait...
     Regenerating the expected 'bpf2c' output...
-    Generating output for atomic_instruction_fetch_add.o
     ...
     ...
-    Generating output for printk_unsafe.o
     Expected 'bpf2c' output regenerated.
     Please verify all the changes then submit the pull-request into the 'release/0.12' branch.
     ```
-1. Verify all the changes then commit all in the working branch.
+4. Verify all the changes then commit all in the working branch.
     >NOTE: The formatting rules may complain about the formatting of the generated `.c` files from the script above. In this case, override them with the following (so they'll work with the `bpf2c_tests` verifying their content):
     >```bash
     >git commit --no-verify
     >```
-1. Create a **Draft** pull-request for your branch into the main repo's "`main`" branch (which you created in step 1), and title the PR as *"Release v`X.Y.Z`"* (replace "`X.Y.Z`" with the version number being released).
-1. Once the CI/CD pipeline for the PR completes, download the "`ebpf-for-windows - MSI installer (Build-x64_Release)`" and "`ebpf-for-windows - NuGet package (Build-x64_Release)`" build artifacts
+5. Create a **Draft** pull-request from your topic branch into the `main` branch of the upstream repo, with the title of the PR as *"Release v`X.Y.0`"* (replace "`X`" and "`Y`" with the version number being released).
+6. Once the CI/CD pipeline for the PR completes, download the "`ebpf-for-windows - MSI installer (Build-x64_Release)`" and "`ebpf-for-windows - NuGet package (Build-x64_Release)`" build artifacts
    (accessible via the "`Actions`" tab on GitHub).
-1. Extract the `*.nupkg` file, and rename it in the following format (replace "`X.Y.Z`" with the version number being released):
+7. Extract the `*.nupkg` file, and rename it in the following format:
+    - `eBPF-for-Windows.X.Y.0.nupkg`
```

Comment:
Why? I don't see a following step actually using this. Nor should there be one. All published binaries should only come from OneBranch builds, never GitHub, and especially not a PR.

Reply:
Fixed.